firehose/firehose.go

```diff
@@ -134,13 +134,37 @@ func newPutRecordBatcher(roleARN, region, firehoseEndpoint, stsEndpoint string)
 		return nil, err
 	}
 
-	svcConfig := &aws.Config{}
+	var svcSess = sess
+	var svcConfig *aws.Config
```

Comment:
https://github.com/aws/amazon-cloudwatch-logs-for-fluent-bit/pull/102/files#r497985487

Reply:
Thanks. Will modify it.